src/sections/cardholders/cardholder-create-widget.tsx

```diff
@@ -1,4 +1,4 @@
-import { faker } from "@faker-js/faker";
+import { allFakers } from "@faker-js/faker";
```

Comment:
this is actually a really large import. I don't love it, but it's necessary to get accurate and realistic autofill test data for all the countries we now support. there's probably a way to reduce this import slightly by _only_ including the locales we need, but even then it would be quite big